index.html

```diff
@@ -426,188 +421,13 @@ <h2>Deciding when to accept an incoming relocation</h2>
 
     </div>
 
-    <h2>Relocated node connection</h2>
```

Comment:
`StartRelocatedNodeConnection` should probably get removed from lines 293-295, too.